### DISCUSSION TITLE
Hide Windows 10 Action Center

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#697] [Q&A] [answered]
I installed ExplorerPatcher and really like it!  Kudos to Val.

Once the Patcher is installed and customized how do we hide the Windows 10 System Tray (Control Center Icon).  I like the gear icon that has been added by the Patcher but don't need to see Windows 10's icon.

![hide this icon](https://user-images.githubusercontent.com/34069523/150279609-c8430bb7-a5ba-4050-956c-6fed49041ee8.png)

**Comment by chip33** [comment, marked as answer]
hi,
The Gear icon is Control Center
To hide Action Center icon.

![Screenshot 2022-01-20 120401](https://user-images.githubusercontent.com/48499740/150336002-390fb1ac-65b4-4505-b387-0fe3266caf66.png)


![Screenshot 2022-01-20 120508](https://user-images.githubusercontent.com/48499740/150336560-b5f49117-3341-4119-b871-78387fff4ace.png)

**Reply by ghost** [reply]
Perfect. Thank you!
